### ISSUE TITLE
BERT - Bidirectional Encoder Representations from Transformers

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#205]
> NLP (Natural Language Processing) 是人工智能（AI）的一个子领域。
# Reference
- [ ] [Github code](https://github.com/google-research/bert)
- [ ] [论文](https://arxiv.org/abs/1810.04805) 
- [ ] [当Bert遇上Keras：这可能是Bert最简单的打开姿势](https://www.jiqizhixin.com/articles/2019-06-21-8)
- [ ] [keras-bert](https://github.com/CyberZHG/keras-bert)
- [ ] [google tensorflow bert代码分析](https://www.cnblogs.com/earendil/p/10465637.html)

# Brief

- 双向 Transformer - 文本序列方向 `左 → 右` + `左 ← 右`
- MLM - Masked LM

**Tasks**
- 分类
- 问答
- 命名实体识别
-

**Comment by junxnone** [comment]
#756
